dashboard/cron.py

```diff
@@ -351,15 +329,31 @@ def weight_consideration(self):
 
         return status
 
+    def update_with_udw_term(self):
+        # cron status
+        status = ""
+
+        logger.debug("in update with udw term")
+
+        # delete all records in the table first
+        status += deleteAllRecordInTable("academic_terms")
+
+        #select file record from UDW
+        term_sql = f"select id, canvas_id, name, date_start, date_end from enrollment_term_dim where date_start > '{settings.EARLIEST_TERM_DATE}'"
+        logger.debug(term_sql)
+        status += util_function(None, term_sql, 'academic_terms')
+
+        return status
+
     def do(self):
         logger.info("************ dashboard cron tab")
 
         status = ""
 
         status += "Start cron at: " +  str(datetime.datetime.now()) + ";"
 
-        logger.info("************ course")
-        status += self.update_with_udw_course()
```

Comment:
Not sure why the course update function was deleted...

Reply:
This was deleted because courses are intended to be populated manually now via the admin UI as part of #305. We didn't need to have a cron to update this table anymore. (Some of this came over from that PR)